### ISSUE TITLE
Tab key no longer expands shortcuts

### ISSUE THREAD

**Issue by smlombardi** [emmetio/emmet-atom#274]
I have been using emmet-atom for a while, and tab expansion worked fine.  After the .200 update, it no longer works.  See attached for trace.  Ctrl+E works.

![2015-05-20_12-35-36](https://cloud.githubusercontent.com/assets/835148/7731222/f7d3446e-feec-11e4-852c-52fe10beb2af.jpg)

**Comment by sergeche** [comment]
Try to switch syntax from `HTML (Angular)` to `HTML`

**Comment by smlombardi** [comment]
That does indeed work.

Of course, as an Angular developer I'll be leaving it on HTML (Angular) for
the improved syntax coloring.

I hope it's a fairly simple fix to add the new type?

**Comment by kukac7** [comment]
:+1:

**Comment by rustedwolf** [comment]
Same on my 0.200 build. 
I have a question reguarding the default key combos are fighting the default atom key combos?
For instance the <kbd>ctrl</kbd>+<kbd>e</kbd> - is a "Find in current buffer", which makes the command usless after install.

**Comment by smlombardi** [comment]
ctrl-E works for expansion.  COMMAND-E adds selected to current buffer.

I'm still hoping for a fix though.

**Comment by rustedwolf** [comment]
"ctrl-E works for expansion.  COMMAND-E adds selected to current buffer."

Well, on my Linux build <kbd>Ctrl</kbd>+<kbd>e</kbd> opens a search in current buffer, and I don't remember having a <kbd>Cmd</kbd> button on Linux. Therefore there is a conflict between emmet's ```emmet:expand-abbreviation``` and atom's default ```find-and-replace:use-selection-as-find-pattern``` combos since the beginning.

**Comment by smlombardi** [comment]
Agreed. There have been a few such conflicts. I've managed to work around a
few editing the key map.

**Comment by oskarrough** [comment]
This is the issue https://atom.io/packages/emmet-simplified (tries?) to solve.

**Comment by sgtpepper43** [comment]
Back to the original issue, Emmet should work regardless of what subtype of html syntax hilighting we're using. If I'm using `HTML (Go)`,`HTML (Angular)`,`HTML (Rails)`,`HTML (Mustache)`, whatever, it should still work. Pull Request #261 has been sitting there for over a month and fixes this problem for `.erb` files. That, along with the current support for `jade`, is way more complicated than simply supporting different syntax hilighting patterns.

**Comment by smlombardi** [comment]
Is there going to be a fix?  It's clearly not working because of the next "file type"

**Comment by trabuccocampos** [comment]
had the same issue in PHP but added
```
atom-text-editor[data-grammar~="php"]:not([mini])':
  'tab': 'emmet:expand-abbreviation-with-tab'
```
to the keymap file shortcuts and it works!
Thanks! (y)

**Comment by iNaD** [comment]
Maybe the global setting of emmet should also be set to `data-grammar~="html"`, because this also allows `HTML (Mustache)`, `HTML (Angular)` etc. to work.

**Comment by sergeche** [comment]
The problem of `data-grammar~="html"` selector is that it will also match other languages as PHP (it’s grammar is `text html php`) and some other languages. 

The ideal solution is to make Tab expander detect common HTML patterns and inner scopes, I plan to add this feature in next release

**Comment by guibot17** [comment]
Hey guys, there is an even simpler solution. In your Atom editor, click on File > Keymap...
This will open the keymap.cson file. Then scroll down, and copy and paste this code:

`'atom-text-editor:not([mini])':
  'ctrl-e': 'emmet:expand-abbreviation'`

Then press enter, and save the file. You're all set.
Happy coding!

**Comment by guibot17** [comment]
Hey guys, there is an even simpler solution. In your Atom editor, click on File > Keymap...
This will open the keymap.cson file. Then scroll down, and copy and paste this code:

`'atom-text-editor:not([mini])':`
`  'ctrl-e': 'emmet:expand-abbreviation'`

Then press enter, and save the file. You're all set.
Happy coding!

**Comment by guibot17** [comment]
Hey guys, there is an even easier solution.
In your Atom editor, click on File > Keymap...
This will open the `keymap.cson file`. Scroll down, and then copy and paste  this code:

`'atom-text-editor:not([mini])':`
    `'ctrl-e': 'emmet:expand-abbreviation'`

Then press enter and save the file. That's it, you're all set!
Happy coding!
